zip-draft-XXXX.rst

```diff
@@ -118,10 +119,44 @@ ecosystem.
 Requirements
 ============
 
-{Describe design constraints on, or goals for the solution -- typically one
-paragraph for each constraint or goal. Again, don't actually specify anything
-here; this section is primarily for use as a consistency check that what is
-specified meets the requirements.}
+1. **Decentralized Funding Mechanism**: The new funding model must implement a
+   decentralized mechanism for disbursing funds. This mechanism should resemble
+   a DAO, ensuring that the community has a direct say in funding decisions.
+   This is crucial to avoid centralization and ensure transparency and fairness
+   in the allocation of resources.
+
+2. **Regulatory Compliance**: The proposed funding model must be designed to
+   minimize regulatory risks. This involves avoiding direct funding of US-based
+   organizations to prevent potential scrutiny from regulatory bodies such as
+   the SEC. The mechanism should comply with applicable laws and regulations to
+   ensure the long-term sustainability of the funding model.
+
+3. **Efficiency in Funding Allocation**: The new model should focus on funding
+   specific projects rather than organizations. This approach aims to ensure
+   that resources are utilized efficiently and that projects receive the
+   necessary support without accumulating excess funds in organizations.
+
+4. **Simplified Tax Compliance**: The funding mechanism must simplify tax
+   compliance for recipients. This can be achieved by deferring tax liabilities
+   to the recipients, allowing them to handle taxes according to their
+   circumstances, thus avoiding the need for organizations to sell block
+   rewards to cover taxes.
```

Comment:
This is going to be relatively complicated from a legal perspective, and it might not actually result in tax savings depending upon how the entities that ultimately receive the funds are structured - for example, if they're for-profit corporations, it could _increase_ the tax burden. It's unlikely that individual contributors can be readily compensated in this fashion, both due to administrative overhead and because (at least in the U.S.) even an individual contributor would likely need to incorporate as a single-member LLC for liability reasons.

Reply:
Good point, requirements need to be rewritten from the standpoint that this is just to create the "lockbox" and not implement the disbursement mechanism anyways ... new draft incoming ...